src/libpgagroal/prometheus.c

```diff
@@ -687,6 +688,9 @@ home_page(int client_fd)
    data = append(data, "  <h2>pgagroal_failed_servers</h2>\n");
    data = append(data, "  The number of failed servers. Only set if failover is enabled\n");
    data = append(data, "  <p>\n");
+   data = append(data, "  <h2>pgagroal_wait_time</h2>\n");
+   data = append(data, "  The waiting time of previous clients\n");
```

Comment:
Just put "The waiting time of clients" here

Reply:
> Just put "The waiting time of clients" here

Done.